en/mulesoft/modules/ROOT/pages/ms_composer_sage_intacct_reference.adoc

```diff
@@ -0,0 +1,225 @@
+** If you are using role-based permissions, create a new role, assign the required permissions for that role, and then create a Sage Intacct Web Service Account to assign to the role.
+** If you are using subscription-based permissions, create a Web Services Account and assign permissions to the role in the *Subscription* section. +
+Additionally, for quote to cash, enable permissions for the *Order Entry* module.
+
+== Connections
+
+include::_partials/connection-definition.adoc[]
+
+include::_partials/connection-name.adoc[]
+
+Company ID::
+
+The customer's company ID. In the field, enter the ID of your Sage Intacct instance. For more information, see the https://developer.intacct.com/web-services/[Sage Intacct documentation].
+
+User ID::
+
+The user ID that you use to log in to Sage Intacct.
+
+User Password::
+
+The password that you use to log into Sage Intacct.
+
+Entity ID::
+
+The Sage Intacct entity to which you want to connect. To connect at the entity level, enter the *Entity ID*. To connect to the top level of the organization, leave this field blank. For more information, see the https://developer.intacct.com/api/company-console/api-sessions/[Sage Intacct documentation].
+
+== Triggers
+
+include::_partials/trigger-definition.adoc[]
+
+New Customer::
+
+Checks for a new Customer record.
+
+New or Updated Customer::
+
+Checks for new or updated Customer record.
+
+New Order::
+
+Checks for a new Order record.
+
+New or Updated Order::
+
+Checks for a new or updated Order record.
+
+== Actions
+
+include::_partials/action-definition.adoc[]
+
+Create Attachment::
+
+Creates a new Attachment record. In Sage Intacct, attachments are entities that contain files. An attachment object can be empty or have files. You can add files to an attachment when you create the record or you can add attachments later. Provide a value for the following:
+* *Attachment Name*
+* *Folder*
+* When adding files to an attachment, use a list as the source of the files, and specify a value in the following fields:
+** *File Binary Data*
+** *File Extension*
+** *File Name*
+
+Update Attachment::
+
+Updates an Attachment record. When updating an attachment, you can update the attachment properties or add new files from a list. Provide a value for *Attachment ID*.
+
+Get Attachments::
+
+Returns a list of Attachment records. To filter the list, provide a value for *Attachment ID*, *Attachment Name*, or *Folder Name*. To return the entire list, leave the fields blank.
+
+Create Attachment Folder::
+
+Creates a new folder for attachments. Provide a value for *Name*.
+
+Get Attachment Folders::
+
+Returns a list of attachment folders. To filter the list, provide a value for *Folder Name* to filter. To return the entire list, leave the fields blank.
+
+Create Customer::
+
+Create a new Customer record. Provide a value for the following:
```

Comment:
The first word of the other descriptions have an s: Creates, Returns, etc. The descriptions should be consistent either way.

Reply:
Good catch!